airbyte-webapp/src/views/Connector/ConnectorForm/FormRoot.tsx

```diff
@@ -1,25 +1,20 @@
 import { Form, useFormikContext } from "formik";
 import React from "react";
 
-import { Spinner } from "components/ui/Spinner";
-
 import { ConnectorDefinitionSpecification } from "core/domain/connector";
 import { FormBlock } from "core/form/types";
 
 import CreateControls from "./components/CreateControls";
 import EditControls from "./components/EditControls";
 import { FormSection } from "./components/Sections/FormSection";
-import ShowLoadingMessage from "./components/ShowLoadingMessage";
 import { useConnectorForm } from "./connectorFormContext";
-import styles from "./FormRoot.module.scss";
 import { ConnectorFormValues } from "./types";
 
 interface FormRootProps {
   formFields: FormBlock;
   hasSuccess?: boolean;
   isTestConnectionInProgress?: boolean;
   errorMessage?: React.ReactNode;
-  fetchingConnectorError?: Error | null;
   successMessage?: React.ReactNode;
   onRetest?: () => void;
   onStopTestingConnector?: () => void;
```

Comment:
I can't comment on it but I think the `| undefined` on `selectedConnector` prop type (the line below this one) can now be removed, because it is now being passed in from the ConnectorForm in a branch where it is verified to not be undefined. 

This should also allow us to remove the `selectedConnector && ()` around the `<CreateControls>` component near the bottom of this file

Reply:
Good catch, removed that stuff as well!